crates/libs/sys/Cargo.toml

```diff
@@ -11,6 +11,14 @@ readme = "readme.md"
 categories = ["os::windows-apis"]
 exclude = ["features.json"]
 
+[lib]
+# This crate does not contain tests. All tests are in separate crates.
+test = false
+doctest = false
+# This crate does not generate docs because it is too large (rustdoc has problems with very large
+# crates) and because the Windows API is documented elsewhere.
+doc = false
```

Comment:
That doesn't sound right. This crate *does* generate documentation, presumably via the standard publishing process through crates.io (see [here]()https://docs.rs/windows-sys/0.52.0/windows_sys/). Will this still be supported after this change?

Reply:
Tim is of course right. Both crates are used to generated Rust docs.

Reply:
Updated.